helm/prometheus-meta-operator/templates/configmap.yaml

```diff
@@ -38,15 +38,17 @@ data:
         {{- range .Values.prometheus.bastions }}
         - {{ . }}
         {{- end }}
-        evaluationInterval: 60s
+        evaluationInterval: {{ .Values.prometheus.evaluationInterval }}
```

Comment:
`scrapeInterval` and `evaluationInterval` were already configurable by command-line arguments, right?
What is the difference now?

Reply:
Now we can set them via giantswarm/config

Reply:
I can check how we Can overrode thé image but i'm not sure we can

Reply:
So I did not explain properly, sorry I was managing the kids. So the values are overridable from cli-arguments but we configured it in a way that the configmap actually fills out the cli args so values we set here are picked up by cobra (the go cli library) as cli args.

So what we set as values here will be set in main.go.

Regarding the image change, we cannot currently override the image repository but only the registry as per https://github.com/giantswarm/prometheus-meta-operator/blob/41bc5acf00a1500b0e7df4299733dc823f996144/service/controller/resource/monitoring/prometheus/resource.go#L144

The values from Config are injected by the configmap